graylog2-server/src/main/java/org/graylog2/bindings/ServerBindings.java

```diff
@@ -155,6 +156,7 @@ private void bindSingletons() {
         bind(BundleExporterProvider.class).in(Scopes.SINGLETON);
         bind(ClusterStatsModule.class).asEagerSingleton();
         bind(ClusterConfigService.class).to(ClusterConfigServiceImpl.class).asEagerSingleton();
+        bind(GrokPatternRegistry.class).in(Scopes.SINGLETON);
```

Comment:
We should really start to consolidate on how to define singleton bindings (either with `Scopes.SINGLETON`, with the `@Singleton` annotation, or with `asEagerSingleton()`)…

Nothing to change here, though. :wink: